bin/matrix-register.js

```diff
@@ -86,58 +86,68 @@ Matrix.localization.init(Matrix.localesFolder, Matrix.config.locale, function ()
                 description: result.description,
               };
 
-              // fire off worker
-              Matrix.firebase.device.add(deviceObj, events)
+              var duplicateDevices = _.values(Matrix.config.deviceMap).filter(function(d) {
```

Comment:
this is a little confusing, as you start with lodash and end with js. try `Object.values` instead of `_-